DfE.FindInformationAcademiesTrusts/Pages/Trusts/Overview.cshtml

```diff
@@ -0,0 +1,96 @@
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-one-half">
+        <section class="govuk-summary-card" data-testid="trust-summary">
+          <div class="govuk-summary-card__title-wrapper">
+            <h2 class="govuk-summary-card__title" id="trust-information">Trust summary</h2>
+          </div>
+          <div class="govuk-summary-card__content">
+            <dl class="govuk-summary-list">
+              <div class="govuk-summary-list__row">
+                <dt class="govuk-summary-list__key govuk-!-width-one-half">
+                  Total academies
+                </dt>
+                <dd class="govuk-summary-list__value">
+                  4
+                </dd>
+              </div>
+              <div class="govuk-summary-list__row">
+                <dt class="govuk-summary-list__key govuk-!-width-one-half">
+                  Academies in each local authority
+                </dt>
+                <dd class="govuk-summary-list__value">
+                  <p class="govuk-!-margin-bottom-1">3 in North Lincolnshire</p>
+                  <p class="govuk-!-margin-bottom-1">1 in North East Lincolnshire</p>
+                </dd>
+              </div>
+              <div class="govuk-summary-list__row">
+                <dt class="govuk-summary-list__key govuk-!-width-one-half">
+                  Pupil numbers
+                </dt>
+                <dd class="govuk-summary-list__value">
+                  4,896
+                </dd>
+              </div>
+              <div class="govuk-summary-list__row">
+                <dt class="govuk-summary-list__key govuk-!-width-one-half">
+                  Pupil capacity <br>(% full)
+                </dt>
+                <dd class="govuk-summary-list__value">
+                  6,463<br>(76%)
+                </dd>
+              </div>
+            </dl>
+          </div>
+        </section>
+      </div>
+      <div class="govuk-grid-column-one-half">
+        <section class="govuk-summary-card" data-testid="ofsted-ratings">
+          <div class="govuk-summary-card__title-wrapper">
+            <h2 class="govuk-summary-card__title" id="ofsted-ratings">Ofsted ratings</h2>
+          </div>
+          <div class="govuk-summary-card__content">
+            <table class="govuk-table" data-module="moj-sortable-table" aria-describedby="ofsted-ratings">
+              <thead class="govuk-table__head">
+                <tr class="govuk-table__row">
+                  <th scope="col" class="govuk-table__header" aria-sort="ascending">Rating</th>
+                  <th scope="col" class="govuk-table__header govuk-table__header--numeric govuk-!-width-one-third" aria-sort="none">Number of schools</th>
+                </tr>
+              </thead>
+              <tbody class="govuk-table__body">
+                <tr class="govuk-table__row">
+                  <td class="govuk-table__cell" data-sort-value="1">Outstanding</td>
```

Comment:
the DfE frontend stylesheets are setting the font size using pixels, is it possible to override this (in our imports file, or in resets). Will `font-size: inherit` work?